providers/openfeature-sdk-meta_provider/openfeature-sdk-meta_provider.gemspec

```diff
@@ -1,8 +1,10 @@
 # frozen_string_literal: true
 
+require_relative "lib/open_feature/sdk/proivder/version"
```

Comment:
```suggestion

require_relative "lib/open_feature/sdk/provider/version"
```

Reply:
This repo uses `openfeature` and `open_feature` as a folder name in various places. I don't have a preference for what we use but we should stay consistent. @maxveldink any preference?